index.html

```diff
@@ -413,6 +416,30 @@ <h3>
       </li>
     </ul>
   </p>
+  <p>
+    The <dfn>change in contributing factors is substantial</dfn> steps are as follows:
+    <ol>
+      <li>
+        If [=implementation-defined=] low-level hardware metrics that contribute to the
+        system performance and [=current pressure state=] drop below or exceed an
+        [=implementation-defined=] threshold for the [=current pressure state=], return true.
+      </li>
+      <li>
+        Return false.
+      </li>
+    </ol>
+    <aside class="note">
+      The [=change in contributing factors is substantial=] algorithm allows user agents to avoid
+      flip-flopping between two states in certain circumstances. For example, a state might otherwise
```

Comment:
two sequential states? or two states that share a threshold?

Reply:
I'd say that would be an implementation detail and we don't want to constrain that.